app/config/AppConfig.scala

```diff
@@ -62,6 +59,10 @@ class AppConfig @Inject()(
   lazy val classificationHelpUrl = loadConfig("urls.classificationHelp")
   lazy val ecicsToolUrl = loadConfig("urls.ecicsTool")
 
+  lazy val eoriService = loadConfig("urls.eoriService")
```

Comment:
Should I move these (and other "static" urls.xxx values) into it's own "config" class?  E.g. ExternalLinksConfig.

Reply:
I think that can be a good idea, but it's up to you :)